src/rsa/padding.rs

```diff
@@ -150,3 +150,138 @@ impl Verification for PKCS1 {
         })
     }
 }
+
+/// PSS Padding as described in https://tools.ietf.org/html/rfc3447#section-9.1.
+/// Salt length is fixed to be the length of the output of the digest algorithm.
+/// The only mask generating function supported is MGF1.
+/// The size (in bits) of the RSA moduli is assumed to be a multiple of 8.
+pub struct PSS {
+    pub digest_alg: &'static digest::Algorithm,
+}
+
+// Maxmimum supported output length for PSS padding.
+const MAX_OUTPUT_LEN: usize = 4096;
```

Comment:
Where does this number come from? The max modulus size we support is 8192 bits (currently), so I would expect this to be `8192 / 8`. Please add a comment about what this is for.

Maybe we need to move `const MAX_BITS: usize = 8192` out of `RSAParameters::verify` so everything can use it?